examples_and_tutorials/millionaires_problem_example/02_store_secret_party_n.py

```diff
@@ -65,15 +65,15 @@ async def main(args = None):
         secret_bindings.add_input_party(party_name, party_id_n)
         print(f"\n🔗 {party_name} sets bindings so that the secret can be input to a specific program (program_id: {args.program_id}) by a specific party (party_id: {party_id_n})")
 
-        # Create permissions object with default permissions for the current user
+        # Create core_concept_permissions object with default core_concept_permissions for the current user
         permissions = nillion.Permissions.default_for_user(user_id_n)
 
-        # Give compute permissions to Alice so she can use the secret in the specific millionionaires program by program id
+        # Give compute core_concept_permissions to Alice so she can use the secret in the specific millionionaires program by program id
```

Comment:
looks like some comments and descriptions got included in the find and replace of permissions